.drone.yml

```diff
@@ -3,23 +3,23 @@ name: qt-5.12
 
 steps:
 - name: cmake
-  image: nextcloudci/client-5.12:client-5.12-12
+  image: nextcloudci/client-5.12:client-5.12-17
   volumes:
     - name: build
       path: /drone/build
   commands:
     - cd /drone/build
-    - cmake -DCMAKE_C_COMPILER=gcc-10 -DCMAKE_CXX_COMPILER=g++-10 -DCMAKE_BUILD_TYPE=Debug -DBUILD_UPDATER=ON -DBUILD_TESTING=1 -DECM_ENABLE_SANITIZERS=address ../src
```

Comment:
are you sure you need to remove the version suffix as we are installing gcc-10 but not using it ?